src/pages/home/sidebar/ChatSwitcherView.js

```diff
@@ -160,7 +160,14 @@ class ChatSwitcherView extends React.Component {
             .map((report) => {
                 const participants = lodashGet(report, 'participants', []);
                 const isSingleUserDM = participants.length === 1;
+                const isGroupDM = participants.length > 1;
```

Comment:
This variable should not be needed as anything which is not a "single user DM" must be a "group DM"